pkg/confidence/confidence_internal_test.go

```diff
@@ -44,6 +44,15 @@ func TestResolveIntValue(t *testing.T) {
 	assert.Equal(t, int64(40), evalDetails.Value)
 }
 
+func TestResolveIntNullValue(t *testing.T) {
+	client := client(t, templateResponse(), nil)
+	client.PutContext("targeting_key", "user1")
+
+	evalDetails := client.GetIntFlag(context.Background(), "test-flag.integer-key-null", 99)
+
+	assert.Equal(t, int64(99), evalDetails.Value)
```

Comment:
We should add the full coverage of `evalDetails`. The expected results for this specific evaluation should be [this](https://github.com/spotify/confidence-sdk-go/commit/4bcfcaf2aadb82c054b793a4943d5708811bf41b), while in the current state an ErrorCode is reported

Reply:
Oww okok I see what you mean, you are absolutely right 👍

Reply:
I've added this control here and in the other tests

Reply:
Great!